src/SPARQLStore/SparqlDBConnectionProvider.php

```diff
@@ -68,19 +68,24 @@ class SparqlDBConnectionProvider implements DBConnectionProvider {
 	 * @param string|null $updateEndpoint
 	 * @param string|null $dataEndpoint
 	 */
-	public function __construct( $connectorId = null, $defaultGraph = null, $queryEndpoint = null, $updateEndpoint = null, $dataEndpoint = null ) {
-		$this->connectorId = $connectorId;
-
-		if ( $this->connectorId === null ) {
-			$this->connectorId = $GLOBALS['smwgSparqlDatabaseConnector'];
+	public function __construct( 
```

Comment:
Yeah, I was kind of stumped on the right thing to do here. Should I ignore those line length warnings for now?